pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -709,21 +714,23 @@ func newUnschedulablePodsMap(metricRecorder metrics.MetricRecorder) *Unschedulab
 	}
 }
 
-// nominatedPodMap is a structure that stores pods nominated to run on nodes.
+// NominatedPodMap is a structure that stores pods nominated to run on nodes.
```

Comment:
why is this exported? we just need a New function exported that returns PodNominator interface.

Reply:
reverted it to be private.

Reply:
> Errors from golint:
pkg/scheduler/internal/queue/scheduling_queue.go:807:27: exported func NewNominatedPodMap returns unexported type *queue.nominatedPodMap, which can be annoying to use

The latest commit to fix the above golint failure.